sdk/eventhub/azure-eventhub/samples/async_samples/send_and_receive_amqp_annotated_message_async.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python
+
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+"""
+Example to show sending, receiving and parsing amqp annotated message(s) to an Event Hub partition.
```

Comment:
nit: not to a partition as in the code we don't specify a partition to send/receive
```suggestion
Example to show sending, receiving and parsing amqp annotated message(s) to Event Hubs.
```